packages/KSelect/KSelect.vue

```diff
@@ -93,7 +93,7 @@
             <KInput
               :id="selectTextId"
               v-bind="$attrs"
-              v-model="filterStr"
+              :value="filterStr"
```

Comment:
question: why are you removing `v-model` here?

Reply:
We need to emit `query-change` event in this `KInput`'s input event handler, so here I'm breaking `v-model` down to `value` prop and `input` event. In the `input` handler, `filterStr` is also updated, so it should have the same effect as `v-model="filterStr"`